src/Shellbar/Shellbar.js

```diff
@@ -514,6 +518,9 @@ Shellbar.propTypes = {
     profileMenu: PropTypes.array,
     /** Holds `searchInput` [properties](?id=component-api-searchinput--compact&viewMode=docs#properties) */
     searchInput: PropTypes.object,
+    /** Provides a size modifier for product actions only visible on small `--mobile` shellbar `s`, or or product actions only visible on desktop `--desktop
+` screens `m`/`l`/`xl`. */
+    sizeOption: PropTypes.string,
```

Comment:
I think this can be reworded and PropType adjusted to prevent confusion of what the options are—`--mobile` and `--desktop` are more implementation details, but developers only directly provide the size option
```suggestion
    /** Provides a size modifier for product actions only visible on small screens (`s`), or product actions only visible on desktop screens (`m`/`l`/`xl`). */
    sizeOption: PropTypes.oneOf(['s','m','l','xl']),
```

Also, I'm not sure that the size modifier works correctly. It seems that the styles break if I change the size option. 
![Screen Shot 2020-09-22 at 2 37 49 PM](https://user-images.githubusercontent.com/22511993/93940365-34afd080-fce1-11ea-8e94-a59c940f5e3a.png)


Ideally, we'd be able to switch between each size cleanly, e.g. manually inspecting the [fundamental-styles examples](https://deploy-preview-1591--fundamental-styles.netlify.app/?path=/docs/components-shellbar--primary) and switching from `fd-shellbar--s` to `--m`, `--l`, or `--xl` works fine.

Reply:
Sounds good, will reword the example and make propTypes a oneOf. As far as the styles breaking, are you on fd-styles version 0.12.0-rc.48 or higher? My styles seem to work fine with the modifiers.
<img width="1137" alt="Screen Shot 2020-09-22 at 3 06 42 PM" src="https://user-images.githubusercontent.com/21978807/93942675-be619d00-fce5-11ea-8b4f-10e0b3c8a295.png">
<img width="541" alt="Screen Shot 2020-09-22 at 3 07 05 PM" src="https://user-images.githubusercontent.com/21978807/93942681-c1f52400-fce5-11ea-87b4-619fa138621b.png">

Reply:
Resolved offline, forgot to include the `fd-shellbar` class